src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.cs

```diff
@@ -231,7 +231,7 @@ internal override bool IsViewHit()
 
 #if __CROSSRUNTIME__
 		// This may need to be adjusted if/when CanContentRenderOutsideBounds is implemented.
-		private protected override Rect? GetClipRect(bool needsClipToSlot, Rect finalRect, Size maxSize, Thickness margin)
+		private protected override Rect? GetClipRect(bool needsClipToSlot, double visualOffsetX, double visualOffsetY, Rect finalRect, Size maxSize, Thickness margin)
```

Comment:
As we are already using `Rect` and `Size` structs, I would have use a `Point` here.